apps/backend/src/libs/modules/token/libs/enums/enums.ts

```diff
@@ -0,0 +1,2 @@
+export { JWTAlgorithm } from "./jwt-algorithm.enum.js";
+export { JWTExpirationTime } from "./jwt-expiration-time.enum.js";
```

Comment:
I don't think we need those types

Reply:
What about types in files jwt-algorithm.type.ts and jwt-expiration-type.ts? Those enums are used to construct the types in these files

Reply:
Or we don't need these types too? Would it be better just to use strings in token module's constructor interface and every other place where algoirthm and expiration time are present?